app/models/task.rb

```diff
@@ -20,6 +20,7 @@
 
 class Task < ApplicationRecord
   has_and_belongs_to_many :campaigns
+  has_and_belongs_to_many :events
```

Comment:
I see your point. Let me put things in other terms: is there anything that might change in the code base that will make this change harder to do in the future rather than now? Unless we get really creative (aka crappy coding), changing this now or in the future will be the exact same effort, so I say we leave it as it is